diff -urN coreavc-for-linux.orig/loader/compat/config.h coreavc-for-linux-svn72/loader/compat/config.h
--- coreavc-for-linux.orig/loader/compat/config.h	2008-10-13 16:43:42.000000000 +0200
+++ coreavc-for-linux-svn72/loader/compat/config.h	2008-10-13 17:17:33.000000000 +0200
@@ -16,7 +16,7 @@
 #define WIN32_LOADER 1
 /* Win32 DLL support */
 #define USE_WIN32DLL 1
-#define WIN32_PATH "/usr/lib/win32"
+#define WIN32_PATH "/usr/lib/codecs"
 /* DirectShow support */
 #define USE_DIRECTSHOW 1
 
diff -urN coreavc-for-linux.orig/loader/elfdll.c coreavc-for-linux-svn72/loader/elfdll.c
--- coreavc-for-linux.orig/loader/elfdll.c	2008-10-13 16:43:43.000000000 +0200
+++ coreavc-for-linux-svn72/loader/elfdll.c	2008-10-13 17:18:12.000000000 +0200
@@ -38,7 +38,7 @@
 extern void dump_exports(HMODULE hModule);
 /*---------------- END HACKS ---------------*/
 
-//char *extra_ld_library_path = "/usr/lib/win32";
+//char *extra_ld_library_path = "/usr/lib/codecs";
 extern char* def_path;
 
 struct elfdll_image
diff -urN coreavc-for-linux.orig/loader/module.c coreavc-for-linux-svn72/loader/module.c
--- coreavc-for-linux.orig/loader/module.c	2008-10-13 16:43:43.000000000 +0200
+++ coreavc-for-linux-svn72/loader/module.c	2008-10-13 17:18:12.000000000 +0200
@@ -365,7 +365,7 @@
 HMODULE WINAPI LoadLibraryExA(LPCSTR libname, HANDLE hfile, DWORD flags)
 {
 	WINE_MODREF *wm = 0;
-	char* listpath[] = { "", "", "/usr/lib/win32", "/usr/local/lib/win32", 0 };
+	char* listpath[] = { "", "", "/usr/lib/codecs", "/usr/local/lib/codecs", 0 };
 	extern char* def_path;
 	char path[512];
 	char checked[2000];
diff -urN coreavc-for-linux.orig/man/dshowserver.1 coreavc-for-linux-svn72/man/dshowserver.1
--- coreavc-for-linux.orig/man/dshowserver.1	2008-10-13 16:43:44.000000000 +0200
+++ coreavc-for-linux-svn72/man/dshowserver.1	2008-10-13 17:19:08.000000000 +0200
@@ -38,7 +38,7 @@
 .SH OPTIONS
 .IP "-c codec_name"
 specify codec to use (normally CoreAVCDecoder.ax).  This
-codec should be installed in /usr/lib/win32
+codec should be installed in /usr/lib/codecs
 .IP "-s <width>x<height>"
 Dimensions of video.  This is used to define buffer sizes
 So it must be the maximum resolution that will be displayed
