diff -urN dshowserver.orig/dshowserver/Makefile dshowserver.patched/dshowserver/Makefile
--- dshowserver.orig/dshowserver/Makefile	2010-01-09 13:33:17.000000000 +0100
+++ dshowserver.patched/dshowserver/Makefile	2010-01-09 16:00:46.301806057 +0100
@@ -2,13 +2,13 @@
 
 
 CFLAGS=-I../loader -Wdisabled-optimization -Wno-pointer-sign -Wdeclaration-after-statement -I. -Wall -Wno-switch -Wpointer-arith -Wredundant-decls -pipe -ffast-math -fomit-frame-pointer -D_LARGEFILE_SOURCE -D__WINE__ -DNOAVIFILE_HEADERS -DMPLAYER
-CFLAGS+=-fno-omit-frame-pointer -mno-omit-leaf-frame-pointer
+OPTFLAGS:=-fno-omit-frame-pointer -mno-omit-leaf-frame-pointer
 
 ifdef DEBUG
-  CFLAGS+= -g
+  OPTFLAGS+= -g
   LOADER_OPT = "DEBUG=1"
 else
-  CFLAGS+= -O2
+  OPTFLAGS+= -O2
 endif
 ifdef STATIC
   STATICBUILD= -static
@@ -28,6 +28,7 @@
   endif
 endif
 
+CFLAGS += $(OPTFLAGS)
 LIBS += $(EXTRA_LIBS)
 
 .PHONY: loader
@@ -35,7 +36,7 @@
 all: $(OBJDIR) dshowserver $(REGISTERCODEC)
 
 dshowserver: $(OBJDIR)/server.o $(OBJDIR)/timeout_sem.o $(OBJDIR)/defaults.o $(OBJDIR)/crc32.o $(OBJDIR)/libloader.a
-	$(CC) $(STATICBUILD) -o $@ $^ $(LIBS)
+	$(CC) $(CFLAGS) $(LDFLAGS) $(STATICBUILD) -o $@ $^ $(LIBS)
 ifdef X_COMPILE
 	mv $@ $@.exe
 	echo "#!/bin/sh" > $@
@@ -45,7 +46,7 @@
 endif
 
 registercodec: ../loader/modify_reg.c ../loader/registry.c
-	$(CC) $(STATICBUILD) -I../loader -I../loader/compat -o $@ $<
+	$(CC) $(CFLAGS) $(LDFLAGS) $(STATICBUILD) -I../loader -I../loader/compat -o $@ $<
 
 
 $(OBJDIR):
